test/createTest.m

```diff
@@ -50,7 +50,8 @@ function createTest()
         if strcmp(jsonFileNameList{i}, 'ex_spm_default')
             jsonLocation = 'http://neurovault.org/collections/1692/ex_spm_default.nidm.zip';
         end
-        %In case this is a json we cannot test, don't make a test for it.
+        %In case this is a json we cannot test, as we cannot obtain the 
```

Comment:
This was a use case for when the json was seperate to the files. It made it easier as we could simply give the test a list of jsons and it would download any missing data for the ones it could. However, a few json files were missing download links so this just handled the case for when there was no data and no link for it to be downloaded from.

This will be removed when we have the viewer working for NIDM zip files rather than jsons.